boolean-uk/api-express-database
# Automate tests with supertst and GH Actions

- add supertest
- add specs based on OPEN API specs
- move api docs into docs
- add scripts to clean database before each test example
- codify publicationDate with quotes
- ensure connection to database through node-postgress is ended after each transaction
